File: pset1/hello/hello.c
#include <cs50.h>
#include <stdio.h>

/*
To be compiled in the cs50 IDE. Sign in with your GitHub account at:

https://ide.cs50.io/

Challenge:

Modify this program in such a way that it first prompts the user for
their name and then prints hello, so-and-so, where so-and-so is their
actual name.
*/

// I'm a function that doesn't return anything!
int main(void)
{
    // Gnab the user's name.
    string name = get_string("What is your name?\n");
    // Print away!
    printf("hello, %s\n", name);
}

File: pset6/hello/hello.py
"""
CS50 Problem Set 6: Hello
-------------------------
Implement a program that prints out a simple greeting to the user, per
the below.

$ python hello.py
What is your name?
David
hello, David
"""


def main():
    """Asks a user for their name. Prints the name, and then prints a
    hello message.
    """
    # Get the name.
    name = input("What is your name? ")
    # Here's the name.
    print(name)
    # Hello!
    print(f"hello, {name}")


if __name__ == "__main__":
    main()
